source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -16,13 +16,20 @@ namespace system {
 namespace {
 
 const auto PXI_5663E = "5663E";
+const int kRisingEdgeDetectionFailedWarning = 379206;
```

Comment:
I"m still trying to figure this out, but the TXP related tests gave this warning but still got results. The full message was "379206: Rising edge detection failed for the acquired waveform.". I'm trying to figure out if this is problematic or not.

Reply:
I think it's OK. I've hit the same type of warning running on hardware that's not set up with the right cables/external-config etc.

Also can you use `UPPER_SNAKE` for constants, at least in files that don't already use `kWhatever`. I know I've been ignoring the session tests ...